### ISSUE TITLE
binance signin requests issue

### ISSUE THREAD

**Issue by mqu** [Jakenberg/binance-ruby#1]
hello Jake,

I'm trying to use binance-ruby.  Running : pp Binance::Api::Account.info! I get an issue about reqest signing : 

```
/var/lib/gems/2.3.0/gems/binance-ruby-0.1.4/lib/binance/api/request.rb:47:in `process!': Binance::Api::Error (-1100): Illegal characters found in parameter 'signature'; legal range is '^[A-Fa-f0-9]{64}$'. (Binance::Api::Error)
	from /var/lib/gems/2.3.0/gems/binance-ruby-0.1.4/lib/binance/api/request.rb:33:in `send!'
	from /var/lib/gems/2.3.0/gems/binance-ruby-0.1.4/lib/binance/api/account.rb:8:in `info!'
	from binance-ruby.rb:19:in `<main>'
```

- signature looks like this : "dQKOP8YFUgyCqoSH6ZYElBy4ZJe7bdZrpHjYZHTjTSo="
- It is working fin with "binance" gem.
- right now, I am using binance-ruby version "0.1.4".

best regards, Marc.

**Comment by mqu** [comment]
issue fixed : 

```
        def signed_request_signature(payload:)
          key = ENV['BINANCE_SECRET_KEY']
          raise Error.new(message: "environment variable 'BINANCE_SECRET_KEY' is required "\
            "for signed requests.") unless key
          # Base64.encode64(OpenSSL::HMAC.digest(OpenSSL::Digest.new('sha256'), key, payload)).strip()
          OpenSSL::HMAC.hexdigest(OpenSSL::Digest.new('sha256'), key, payload)
        end
```

also, we need to change timestamp to : 

```
# in configuration.rb and 
require 'date'
...
        def timestamp
          DateTime.now.strftime('%Q')
        end
...
# request.rb
        def timestamp
          Configuration.timestamp
        end
```

**Comment by Jakenberg** [comment]
Hi @mqu, thanks for using this gem! This indeed was a bug: fixed in [0.1.5](https://github.com/Jakenberg/binance-ruby/releases/tag/0.1.5). 🚀
